src/streamlit_app.py

```diff
@@ -25,9 +26,11 @@
     }
 )
 
+image1 = Image.open('../src/assets/profile-with-readme.png')
```

Comment:
⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?